var/spack/repos/builtin/packages/py-opppy/package.py

```diff
@@ -12,10 +12,11 @@ class PyOpppy(PythonPackage):
     output and dump files generated by scientific software packages."""
 
     homepage = "https://github.com/lanl/opppy"
-    url = "https://github.com/lanl/OPPPY/archive/OPPPY-0_1_1.tar.gz"
+    url = "https://github.com/lanl/OPPPY/archive/opppy-0_1_2.tar.gz"
     git = "https://github.com/lanl/opppy.git"
 
     version('master', branch='master')
+    version('0_1_2', sha256='ef3795d3164fa0aa7ea7da7e223d6d0a48d2960aefd03a7d90cdb8b8f480cd4c')
     version('0_1_1', sha256='505c023853e75552abc65de9777a125ecb6a99a1cb4e605a4f702af837e3168b')
 
     depends_on('py-setuptools', type=('build', 'run'))
```

Comment:
Is setuptools actually needed at runtime? Also, it looks like sphinx is only needed if you build the docs, so that dependency can be dropped.

Would you like to add yourself as a maintainer of the package? Just add `maintainers = ['clevelam']` and we'll ping you if anyone else submits a PR or issue related to this package.